### ISSUE TITLE
fine tuning question

### ISSUE THREAD

**Issue by DecentMakeover** [flairNLP/flair#976]
A clear and concise description of what you want to know.
I have trained a model using ELMO embeddings and it is performing well, I have a saved model now.

Is there a way for me to fine tune this model using bert embeddings?

Thanks

**Comment by TheArowanaDude** [comment]
I have a similar question as well; is there a way to fine tune the hyper-parameters of a model I already trained? I noticed that Flair does some form of optimization of the learning rate, does this optimization extend to the other hyperparameters?

**Comment by BernierCR** [comment]
Flair included Hypopt, which does a grid search I think. Or some kind search at least. 

You list the parameters / values you want to search. At the end, it tells you which combination is best. Then you just retrain a model using those parameters. 

The caveat is that this only works with one GPU (or zero GPUs). If you want to hyper parameter tune on a GPU cluster, you will need another method. 

and Bert embeddings are supported. It's easy try bert instead of Elmo. You could just try it, switch manually yourself, or you could use hyperopt / flair integration to search the embedding space.
